### PR TITLE
Style comment card

### PR DESCRIPTION
**Pull request by NeuralAlchemist**
Added basic styling for content, user name, delete & edit icons and for outline of comment card
Screenshot 2021-05-11 at 10 07 00
Screenshot 2021-05-11 at 10 06 48

### CODE REVIEW

File: frontend/src/components/comments/CommentCard.jsx [hunk 1]
```diff
@@ -8,101 +8,121 @@ import CommentUpdateForm from "./CommentUpdateForm";
...
+                        </span>
+                    )}
+                </div>
+                <Moment className="time-lapse" fromNow>
+                    {comment.createdTime}
+                </Moment>
+            </div>
+            <div className="word-wrap comment-content">
+                {!toggle ? (
+                    comment.contentText
```
Comment (eeels22) [inline frontend/src/components/comments/CommentCard.jsx]: I like what you did with the toggling the view of the contentText or update form!

Perhaps we could wrap `{comment.contentText}` in `<p>` tags so it inherits the same font family as post contentText?

File: frontend/src/components/comments/CommentCard.jsx [hunk 2]
```diff
@@ -8,101 +8,121 @@ import CommentUpdateForm from "./CommentUpdateForm";
 import CommentLikeApi from "../../api/CommentLikeApi";
 
 export default function CommentCard({
-  postId,
-  comment,
-  onDeleteClick,
-  currentUser,
+    postId,
```
Comment (eeels22) [inline frontend/src/components/comments/CommentCard.jsx]: Pls could we run the file through VSC formatter?

**Review by eeels22 — CHANGES_REQUESTED** [review]
Looking good Yash! Just a suggestion to wrap comment contentText  in <p> to inherit the same styling as post contentText

File: frontend/src/components/comments/CommentCard.jsx (hunk 1 above)
Comment (NeuralAlchemist) [reply frontend/src/components/comments/CommentCard.jsx]: Done 
Screenshot 2021-05-11 at 13 40 34

File: frontend/src/components/comments/CommentCard.jsx (hunk 2 above)
Comment (NeuralAlchemist) [reply frontend/src/components/comments/CommentCard.jsx]: i think it has been updated now please check and let me know!

**Review by janko101 — APPROVED** [review]
It looks good to me, when En-Chi approves we can MMMerggge :)

**Review by eeels22 — APPROVED** [review]
Great work Yash!

**Pull request merged**